.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,44 @@
+name: Test Windows
+
+on:  [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [python3]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Just to clarify, this just sets some environmental variables so the msvc tools are directly available. Does Faber need this to have happened? B2 didn't use this method (though it could fall back to it). 

This might be the way to go for this use case. It also looks like you can set 64 vs. 32 bit builds here...though we'd also have to get the correct python installed. The [image docs](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#python) say both x86 and x64 are on there, but I'm not clear where they are.

Reply:
I see. No, Faber only needs to be able to *find* MSVC, using `vswhere`. So as long as MSVC and `vswhere` are installed, all is fine. If `ilammy/msvc-dev-cmd` only sets some variables, it's of no use here indeed and I will remove it.